pages/index.js

```diff
@@ -53,15 +55,18 @@ export function getServerSideProps(context) {
   }
 
   return {
-    props: {}
+    props: {},
+    redirect: {
+      destination: config.WIX_URL,
+    },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a redirect to `config.WIX_URL` is a significant change. Ensure that this URL is valid and accessible, and that this redirection aligns with the expected user flow. If `WIX_URL` is not defined, this could result in a redirect to `undefined`, which would likely cause an error. It might be beneficial to add a fallback URL or handle the case where `WIX_URL` is not defined.

```diff
- destination: config.WIX_URL,
+ destination: config.WIX_URL || '/fallback-url',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->